site/docs/v1/tech/tutorials/switch-search-engines.adoc

```diff
@@ -0,0 +1,89 @@
+---
+layout: doc
+title: Switching search engines
+description: How to switch back and forth between the provided search engines
+---
+
+As of version 1.16, FusionAuth supports two search engines:
+
+* database
+* elasticsearch
+
+and you can easily switch between them. The strengths and limitations of each search engine type link:/docs/v1/tech/core-concepts/users#user-search[are documented here].
+
+This tutorial walks you through changing your search engine. Reasons for modifying it vary; you may be simplifying your deployment strategy by using the database search engine. Or perhaps you need the advanced searching options of Elasticsearch. Either way, it's an easy switch.
+
+=== Prerequisites
+
+This tutorial assumes you have both the application and the search services running on Linux. You also must be running at least version 1.16.0. 
+
+If you don't, head on over to the link:/docs/v1/tech/installation-guide/fusionauth-search[installation guide]. If you already have the application running, but need to install the search service, link:/docs/v1/tech/installation-guide/fusionauth-search[here are instructions].
+
+If you are running FusionAuth via Docker, a Windows service, or some other mechanism, you'll have to stop and start FusionAuth in an installation specific manner.
+
+=== From Elasticsearch to Database
+
+Stop FusionAuth:
+
+```
+sudo service fusionauth-app stop
+sudo service fusionauth-search stop
+```
+
+Locate your FusionAuth configuration file, it might be at a location similar to `/usr/local/fusionauth/config/fusionauth.properties`.
+
+Comment out or remove the `fusionauth-search.*` properties and the `fusionauth-app.search-servers` value. 
+
+Add or update the `fusionauth-app.search-engine-type` key to have the value `database`.
+```
+fusionauth-app.search-engine-type=database
```

Comment:
This is actually the only change necessary, the other values will be ignored if we are not using Elasticsearch. There may be some value to leave the existing values as is - in case you switch back. 

```
fusionauth-app.search-engine-type=database
```

You could also set this by using the environment variable `FUSIONAUTH_SEARCH_ENGINE_TYPE`

https://fusionauth.io/docs/v1/tech/reference/configuration

Reply:
awesome, good feedback.